packages/api/src/command/medical/patient/get-hl7v2-subscribers.ts

```diff
@@ -53,7 +53,7 @@ export async function getHl7v2Subscribers({
   states,
   subscriptions,
   pagination,
-}: GetHl7v2SubscribersParams): Promise<Hl7v2Subscriber[]> {
+}: GetHl7v2SubscribersParams): Promise<PatientModelReadOnly[]> {
```

Comment:
Why are we returning a Model here? Especially because we use the return of this function to send over the wire, this should be the domain, obtained from `.dataValues()`, right?